seattle_example/seattle_tasks.yml

```diff
@@ -69,18 +72,24 @@
 #
 - name: Ingest census blocks to postgres
   command: ogr2ogr -f PostgreSQL -t_srs EPSG:4326 -nln seattle_blkgrpce10 -nlt MultiPolygon 'PG:user=postgres dbname=piecewise' /opt/piecewise_web/seattle_blkgrpce10/CENSUS10_blkgrp_WGS.shp
```

Comment:
You don't have to do this, but to keep line lengths reasonable (with 80 chars?) YAML (and hence Ansible) has a facility called ["folded" scalars](http://www.yaml.org/spec/current.html#id2503232), that let's you wrap long lines, and the output is the lines concatenated, separated by a single space... like this:

```
command: >
  ogr2ogr -f PostgreSQL -t_srs EPSG:4326 -nln seattle_blkgrpce10
  -nlt MultiPolygon 'PG:user=postgres dbname=piecewise'
  /opt/piecewise_web/seattle_blkgrpce10/CENSUS10_blkgrp_WGS.shp